testing/src/test/java/com/baeldung/lambdabehave/CalculatorTest.java

```diff
@@ -0,0 +1,53 @@
+package com.baeldung.lambdabehave;
+
+import com.insightfullogic.lambdabehave.JunitSuiteRunner;
+import com.insightfullogic.lambdabehave.Suite;
+import com.insightfullogic.lambdabehave.generators.Generator;
+import com.insightfullogic.lambdabehave.generators.SourceGenerator;
+import org.junit.runner.RunWith;
+
+@RunWith(JunitSuiteRunner.class)
+public class CalculatorTest {
+
+    private Calculator calculator;
+
+    {
+        Suite.describe("Lambda behave example tests", it -> {
+
+            it.isSetupWith(() -> {
+                calculator = new Calculator(1, 2);
+            });
+            it.should("Add the given numbers", expect -> {
+                expect.that(calculator.add()).is(3);
+            });
+            it.should("Throw an exception if divide by 0", expect -> {
+                expect.exception(ArithmeticException.class, () -> {
+                    calculator.divide(1, 0);
+                });
+            });
+            it.uses(2, 3, 5)
+                    .and(23, 10, 33)
```

Comment:
Please reformat files using our classic formatter. The continuation indentation should be equal to 2 spaces.